virtualbox.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+from pathlib import Path
+from sys import platform
+import datetime
+from pytz import UTC
+
+utc = UTC
+
+class virtualbox(object):
+
+    def __init__(self, virtualboxpath, tz) -> None:
+        if platform.lower() == "win32":
+            virtualboxpath = virtualboxpath.replace('\"', "")
+            virtualboxpath = virtualboxpath.replace("\'", "")
+            self.vmrunpath = Path(virtualboxpath).joinpath("VBoxManage.exe")
+        else:
+            self.vmrunpath = virtualboxpath
+        self.tz = tz
+        self.vminfo = []
+        self.output = []
+
+    def updateOutput(self) -> None:
```

Comment:
had the feeling, that a lot of zombie processes were eating up my memory so I tried context managers as a counter measure